src/main/java/com/dreammaster/gthandler/recipes/ElectrolyzerRecipes.java

```diff
@@ -8,293 +8,491 @@
+                GT_Values.NI,
                 GT_Values.NF,
-                Materials.Fluorine.getGas(2000L),
-                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Potassium, 1L),
-                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Magnesium, 3L),
-                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminiumoxide, 3L),
-                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.SiliconDioxide, 3L),
-                GT_OreDictUnificator.get(OrePrefixes.cell, Materials.Oxygen, 4L),
+                Materials.Oxygen.getFluid(1000L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Potassium, 2L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminiumoxide, 5L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.SiliconDioxide, 18L),
+                GT_Values.NI,
+                GT_Values.NI,
+                GT_Values.NI,
+                null,
+                260,
+                90);
+        // Biotite
+        GT_Values.RA.addElectrolyzerRecipe(
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Biotite, 45L),
+                GT_Values.NI,
+                GT_Values.NF,
+                Materials.Fluorine.getGas(4000L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Potassium, 2L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Magnesium, 6L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminiumoxide, 15L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.SiliconDioxide, 18L),
+                GT_Values.NI,
                 GT_Values.NI,
                 null,
                 280,
                 120);
+        // Red Granite
         GT_Values.RA.addElectrolyzerRecipe(
-                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Lazurite, 14L),
-                GT_Utility.getIntegratedCircuit(1),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.GraniteRed, 6L),
+                GT_Values.NI,
                 GT_Values.NF,
                 GT_Values.NF,
-                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminiumoxide, 3L),
-                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.SiliconDioxide, 3L),
-                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Calcium, 4L),
-                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Sodium, 4L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.PotassiumFeldspar, 1L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminiumoxide, 5L),
+                GT_Values.NI,
+                GT_Values.NI,
                 GT_Values.NI,
                 GT_Values.NI,
                 null,
-                364,
+                120,
+                60);
+        // Pollucite
+        GT_Values.RA.addElectrolyzerRecipe(
```

Comment:
the total is incorrect. should be just 22. you should also keep the water intact imo.

Reply:
I'm pretty sure 26 is the right number, I made the water into hydrogen and oxygen because these changes will overall nerf the amount of oxygen you get from ores, so this can partially compensate for that (when you get MV electrolyzer it gets back to normal though)

Reply:
its 2 Cs + 2Al + 4Si + 2 H2O + 12 O. so 22 in total, if you split up the water or not.
and its specifically H2O in here not seperate H and O, so keeping the compounds makes a lot more sense.

Reply:
You're absolutely right. I have no idea why, but I always apply the rule correctly when it comes to oxides, but for some reason I just assume that 2 water is just 2 water and not 2/3. I'll fix that

Reply:
well 2 water is 4H + 2O. but its not 6 water. It gets fun when the fluid and solid rules are nested :P

Reply:
I'm not sure what to do with this now tbh, I'd have to multiply the recipe a lot

Reply:
22 pollucite -> 2 cs + 5alumina + 12 silicondioxide + 2 water + 1O

Reply:
done